packages/engine/Specs/Scene/VoxelPrimitiveSpec.js

```diff
@@ -83,6 +83,9 @@ describe(
       });
       expect(spyUpdate1.calls.count()).toEqual(1);
       expect(spyUpdate2.calls.count()).toEqual(1);
+      expect(primitive.statistics.numberOfTilesWithContentReady).toEqual(1);
```

Comment:
Can we update the name/description of this spec to indicate that we are checking more than just "initial tiles loaded and all tiles loaded events" ?